tests/integration/test_requests.py

```diff
@@ -114,7 +114,7 @@ def test_post_chunked_binary(tmpdir, httpbin):
     assert req1 == req2
 
 
-@pytest.mark.xfail('sys.version_info >= (3, 6)', strict=True, raises=ConnectionError)
+@pytest.mark.xfail('sys.version_info >= (3, 5)', strict=True, raises=ConnectionError)
```

Comment:
`pytest.mark.xfail(sys.version_info >= (3, 6) and platform.python_implementation() == 'CPython', strict=True, ...)`

Reply:
because it works fine on pypy